parsons/databases/postgres/postgres.py

```diff
@@ -52,7 +54,13 @@ def __init__(
         self.timeout = timeout
         self.dialect = "postgres"
 
-    def copy(self, tbl, table_name, if_exists="fail", strict_length=False):
+    def copy(
+        self,
+        tbl: Table,
+        table_name: str,
+        if_exists: str = "fail",
+        strict_length: bool = True,
```

Comment:
This could theoretically be a breaking change for some pipelines. To @elyse-weiss comment about improving padding defaults, what if we added `padding` to the ABC? I think a sensible default padding makes sense to me and would mean this would be less likely to break things

Reply:
Yes, that's true. I think at some point we're probably going to want to make a breaking change. For the meantime, I've reverted this change.

Regarding padding, I think that's a good idea. We should probably add that to the issue tracking the features to add to database parity, and we can work on that once the ABC is merged into `major-release`.

Reply:
@shaunagm We're tracking these changes to do in an issue, right? Could we add padding there?

Reply:
Added it here: https://github.com/move-coop/parsons/issues/838#issuecomment-1682849628